keyboards/kbd6x/keymaps/othi/config.h

```diff
@@ -0,0 +1,9 @@
+#define ONESHOT_TAP_TOOGLE 2
+#define ONESHOT_TIMEOUT 2000
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This should be removed, since this is now the default, and is turned off by defining "STRICT_LAYER_RELEASE`.

```suggestion
// #define STRICT_LAYER_RELEASE
```